### DISCUSSION TITLE
Permission error with NordVPN

### DISCUSSION THREAD

**Discussion by tcoombs** [RaspAP/raspap-webgui#1697] [Q&A] [answered]
I have installed the NordVPN client which works fine if I activate the tunnel using the NordVPN Connect command. I've also enabled NordVPN in the RaspAP installation and so I see this option in the menu. The problem I have is that if I tried to activate a NordVPN tunnel via the Web interface of RaspAP I get the following error: "Unable to execute NordVPN binary found at: /usr/bin/nordvpn". I also see this message: "Check that binary is executable and permissions exist in raspap.sudoers:" 

I have tried sudo usermod -aG nordvpn $USER and rebooted my RP4 but this has not resolved the issue. I'm am currently using RaspAP version 3.2.1. Is there anyone who could give me a few tips on solving this issue? Thanks, Tony

**Comment by billz** [comment, marked as answer]
The NordVPN binary needs execute permissions in `raspap.sudoers` (typically this is added by the installer). Edit it safely with `sudo visudo /etc/sudoers.d/090_raspap`.
```
www-data ALL=(ALL) NOPASSWD:/usr/bin/nordvpn *
```

If you don't see the above at the end of the file, append it as shown. Save the file and exit.

**Reply by tcoombs** [reply]
Thanks Bill for your help and quick reply. That fixed it. Cheers, Tony

**Reply by billz** [reply]
@tcoombs glad to hear it! cheers
